terracotta-kit/pom.xml

```diff
@@ -15,6 +15,9 @@
     <packaging>pom</packaging>
     <name>Terracotta Kit</name>
 
+  <properties>
```

Comment:
Not sure; possibly.  IIRC, I wound up including this property because the source code level determined by IntelliJ IDEA was the default (5. I believe) so I set this here.  That may have been before I included the `maven-compiler-plugin` element.  But, since the module artifact is a pom, it may need to be set directly.

I'll toy with it and see if it's really needed.

Reply:
IntelliJ IDEA is picking up the correct source version now -- I presume it's because of the `maven-compiler-plugin` execution I added (resulting in recognition of the inherited property).  Will remobv